package com.tiviplay.tiviplaybox.model.SbpCombinedResponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import java.util.Map;

/* JADX INFO: loaded from: classes3.dex */
public class GetOvpnzip {

    @SerializedName("access")
    @Expose
    private Map<String, Credential> access;

    @SerializedName("link")
    @Expose
    private String link;

    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("result")
    @Expose
    private String result;

    @SerializedName("sc")
    @Expose
    private String sc;

    @SerializedName("vpnstatus")
    @Expose
    private String vpnstatus;

    public Map<String, Credential> getAccess() {
        return this.access;
    }

    public String getLink() {
        return this.link;
    }

    public String getMessage() {
        return this.message;
    }

    public String getResult() {
        return this.result;
    }

    public String getSc() {
        return this.sc;
    }

    public String getVpnstatus() {
        return this.vpnstatus;
    }

    public void setAccess(Map<String, Credential> map) {
        this.access = map;
    }

    public void setLink(String str) {
        this.link = str;
    }

    public void setMessage(String str) {
        this.message = str;
    }

    public void setResult(String str) {
        this.result = str;
    }

    public void setSc(String str) {
        this.sc = str;
    }

    public void setVpnstatus(String str) {
        this.vpnstatus = str;
    }
}
